### PR TITLE
Ignore metakey in keyboard shortcuts

### PR DESCRIPTION
**Pull request by garyburd**
The F keyboard short blocked access to search in Safari. Fix by ignoring keyboard input when the metakey is pressed.

### CODE REVIEW

File: doc/goprint.go [hunk 1]
```diff
@@ -62,8 +62,8 @@ func main() {
 	c := spew.NewDefaultConfig()
 	c.DisableMethods = true
 	apkg, _ := ast.NewPackage(fset, files, importer, nil)
-	c.Dump(apkg)
-	ast.Print(fset, apkg)
+	//c.Dump(apkg)
```
Comment (adg) [inline doc/goprint.go]: this seems like an unrelated change?

**Comment by garyburd** [comment]
Oops, sorry about that. Please look again.
